README.md

```diff
@@ -3,8 +3,28 @@ Terraform module for [Azure Database for PostgreSQL - Flexible Server](https://d
 
 ## Example
 
+provider.tf
+```hcl
+provider "azurerm" {
+  features {}
+}
+
+provider "azurerm" {
+  features {}
+  skip_provider_registration = true
+  alias                      = "postgres_network"
+  subscription_id            = var.aks_subscription_id
```

Comment:
missing variables.tf for the `aks_subscription_id`

Reply:
some reason didn't push it initially - now added